arrr.py

```diff
@@ -24,66 +24,25 @@
 CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 """
 import argparse as arrrgparse  # Geddit..? ;-)
+from csv import reader as HELMSMAN  # He be the only lad aboard who can read!
 import random
 import sys
 
-
 #: The help text to be shown when requested.
 _HELP_TEXT = """
 Take English words and turn them into something Pirate-ish.
 
 Documentation here: https://arrr.readthedocs.io/en/latest/
 """
 
-
 #: MAJOR, MINOR, RELEASE, STATUS [alpha, beta, final], VERSION
-_VERSION = (1, 0, 3)
-
+_VERSION = (1, 1, 0)
```

Comment:
Your editor be removing all them nice blank lines.  Mates need restin betwen workin, ya know?